partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -261,19 +261,19 @@ rel::{
       ],
     },
 
-    union::{
-      lhs: rel,
-      rhs: rel,
-    },
-
-    intersect::{
-      lhs: rel,
-      rhs: rel,
-    },
-
-    except::{
+    // From Substrait: "The set operation encompasses several set-level operations that support combining datasets, ...
+    //  possibly excluding records based on various types of record level matching."
+    set::{
       lhs: rel,
       rhs: rel,
+      type: [
+        UNION_ALL,
+        UNION_DISTINCT,
+        INTERSECT_ALL,
+        INTERSECT_DISTINCT,
+        EXCEPT_ALL,
+        EXCEPT_DISTINCT
+      ]
```

Comment:
We've gone back and forth for this modeling. I believe I did "set" initially, but PIG modeled as `(union ...) (intersect ...) ...` etc. hence them being their own operators. 

I'm keen on keeping them separate mostly for staying similar to PartiQL 0.14, but this is entirely preference. However, I would like the set quantifier to be split.

```
   union::{
      setq: set_quantifier,
      lhs: rel,
      rhs: rel,
    },

    intersect::{
      setq: set_quantifier,
      lhs: rel,
      rhs: rel,
    },

    except::{
      setq: set_quantifier,
      lhs: rel,
      rhs: rel,
   }
```

Interestingly, substrait uses a list of inputs rather than lhs, rhs. Seems like an optimization